spec/iex/endpoints/historical_prices_spec.rb

```diff
@@ -0,0 +1,170 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe IEX::Resources::HistorialPrices do
+  include_context 'client'
+
+  context 'known symbol' do
+    context 'with valid params' do
+      context 'with defaults', vcr: { cassette_name: 'historical_prices/msft' } do
+        subject do
+          client.historical_prices('MSFT')
+        end
+        let(:historical_price) { subject.first }
+
+        it 'retrieves historical prices' do
+          expect(subject.size).to eq 23
+          expect(historical_price.date).to eq '2020-10-07'
```

Comment:
Should this be converted to a `Date` as well?

Reply:
Yeah.... I think it should be however, all the other endpoints that return dates are also strings. I say lets leave this as is and I'll fix all of them up in a follow up PR.

Reply:
For this, would you prefer two fields, one for the raw data and one for the converted data? Similar to how dollar values are handled?

Reply:
Dollar values are different from dates since those are integers in cents, and formatted values in dollars. I think this should just be a `Date` type. NBD though, I'm going to merge as is and feel free to change it on top before the next release, double-checking that we do it everywhere a date is returned in other instances.